### ISSUE TITLE
When I want to apply measure to door doesn't behave the same with both

### ISSUE THREAD

**Issue by gervasemangwana** [emoncms/MyHomeEnergyPlanner#200]
One takes me to the empty door's measures library the other to windows measures. Even though both are pink highlighted as doors

**Comment by cagabi** [comment]
Can you please share the assessment with me so i can see the error myself

**Comment by gervasemangwana** [comment]
I’ve shared with you Carlos


Gervase Mangwana

Waxwing Energy
07956378981
info@waxwingenergy.co.uk

**Comment by cagabi** [comment]
Thanks Gervase.

So the reason why it was displaying the window measures library is because the element type was _window_ (meaning you didn't change the type when adding it). The reason to be displayed pink is very silly, before i introduced the distinction between windows, doors, rooflights and hatches; the color was picked from the name of the element, so because the word _door_ was in the name of the element the color was pink. I have fixed that now and we only use the type of the element for picking the color. I havve also manually overwritten the type of the DRDT_GM elements in the assessments so they are of the _door_ type.

There is an issue that cannot be changed which is that if there is no measure in the library, when applying a measure the dropdown will show by default _Wall_, it doesn't make a difference and applying the measure will add the correct type of element. But it is confusing.

Basically we need to add Doors, and hatches and rooflights to the fabric measures library
